src/compiler/diagnosticMessages.json

```diff
@@ -1689,6 +1689,10 @@
         "category": "Error",
         "code": 2650
     },
+    "Merged declaration '{0}' cannot include a default export declaration. Consider adding a separate 'export default {0};' declaration.": {

```

Comment:
I would add the word "instead" at the end. Also, for some reason I find the semicolon distracting in the error message. Maybe you should remove it.